src/systems/detachable_joint/DetachableJoint.hh

```diff
@@ -143,6 +143,10 @@ namespace systems
     /// \brief Whether all parameters are valid and the system can proceed
     private: bool validConfig{false};
 
+    /// \brief Set whether the fixed joint should weld the child entity to the
+    /// parent entity.
+    /// No-op if the underlying physics engine does not support the feature.
+    private: bool weldChildToParent{false};
```

Comment:
Please add documentation to the "System parameters" section above. Also mention that this is needed for the bullet physics engine and add some explanation as to why and when it would be needed.

Reply:
added doc for param. [aaeb155](https://github.com/gazebosim/gz-sim/pull/2984/commits/aaeb1550237b554dea54b1b2e3832804638d61a6)